rubix-bookkeeper/pom.xml

```diff
@@ -112,13 +112,19 @@
                             <artifactSet>
                                 <includes>
                                     <include>com.google.guava:guava</include>
+                                    <include>com.readytalk:metrics-statsd-common</include>
+                                    <include>com.readytalk:metrics3-statsd</include>
```

Comment:
@vrajat StatsDReporter dependencies are now being shaded. 
This has also been tested on a Qubole cluster.